app/components/entry/CompactSection/index.tsx

```diff
@@ -194,6 +230,10 @@ function CompactSection(props: Props) {
         onAddButtonClick,
     ]);
 
+    if (emptyMessageHidden && widgetsWithValue?.length === 0) {
+        return null;
+    }
```

Comment:
Why return `null` here? This will impact all CompactSection

Reply:
yes, makes sense